CHANGELOG.md

```diff
@@ -67,6 +67,7 @@
 - Fixed a bug where `Resampler` methods on timedelta columns would produce integer results.
 - Fixed a bug where `pd.to_numeric()` would leave `Timedelta` inputs as `Timedelta` instead of converting them to integers.
 - Fixed `loc` set when setting a single row, or multiple rows, of a DataFrame with a Series value.
+- Fixed a bug where `reindex` failed when the new index is a Series with non-overlapping types from the original index.
```

Comment:
```suggestion
- Fixed a bug where `reindex` fails when the new index is a Series with non-overlapping types from the original index.
```
i think we use present tense for changelogs?

Reply:
I think I saw both but this change is more grammatically correct :)